src/routes/about/+page.svelte

```diff
@@ -1,76 +1,193 @@
-<script lang="ts">
-  import placeholder from '$lib/assets/avatars/avatar-placeholder.png';
-  import TeamMember from '$lib/components/TeamMember.svelte';
-</script>
-
 <head>
   <title>About - KHA</title>
 </head>
 
-<head>
-  <title>About - KHA</title>
-</head>
+<script lang="ts">
+  import placeholder from '$lib/assets/avatars/avatar-placeholder.png';
+  import TeamMember from '$lib/components/TeamMember.svelte';
+  import khaLogo from '$lib/assets/kha-logo.gif';
+  import type { PageServerData } from './$types';
+  import { Carousel, CarouselControl, CarouselItem } from 'sveltestrap';
+  
+  export let data: PageServerData;
```

Comment:
```suggestion
  import type { PageData } from './$types';
  import { Carousel, CarouselControl, CarouselItem } from 'sveltestrap';
  
  export let data: PageData;
```
The `PageData` type is used in the docs and I think it should auto-generate the type (this will be irrelevant if we move it from a load function to an API route)